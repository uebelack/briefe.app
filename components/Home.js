import React from 'react';
import Image from 'next/image';
import getLocale from '@/functions/getLocale';
import getTranslations from '@/functions/getTranslations';
import Container from './Container';
import Feature from './Feature';
import features from '@/data/features';
import Heading from './Heading';
import Page from './Page';
import ScrollButton from './ScrollButton';
import Section from './Section';
import Teaser from './Teaser';
import Blog from './Blog';
import getBlogArticles from '@/functions/getBlogArticles';
import ContactForm from './ContactForm';

export default function Home() {
  const locale = getLocale();
  const t = getTranslations();

  const featuresKey = Object.keys(features).find((key) => key.startsWith(locale ?? 'en-US')) ?? 'en-US';
  const featureData = features[featuresKey];
  const articles = getBlogArticles(locale);

  return (
    <Page>
      <Section className="grid place-items-center first bg-blue text-white">
        <Container className="grid place-items-center">
          <Image src={`/devices_${locale?.startsWith('de') ? 'de' : 'en'}.png`} alt="Letter app" width={992} height={605} unoptimized />
          <a href="http://itunes.apple.com/app/letter/id498506154">
            <Image src="/app_store_en.svg" alt="App Store" width={150} height={50} className="mt-5" />
          </a>
          <div className="p-3 md:w-4/5 pt-10">
            <Teaser text={t('home.teaser')} />
          </div>
        </Container>
      </Section>
      <ScrollButton label={t('common.scroll_down')} target="details" />
      <Section>
        <Container>
          <div className="flex">
            <div className="prose proese-briefe">
              <Heading id="details">{t('details.title')}</Heading>
              <p>
                {t('details.p1')}
              </p>
              <p>
                {t('details.p2')}
              </p>
              <p>
                {t('details.p3')}
              </p>
            </div>
            <div className="pt-32 ml-20 hidden lg:inline">
              <Image src={`/details_${locale === 'de' ? 'de' : 'en'}.png`} alt="Letter app" width={225.5} height={450} unoptimized />
            </div>
          </div>
        </Container>
        <Container>
          <Heading id="features">{t('features.title')}</Heading>
          <p>{t('features.description')}</p>
          <div className="flex flex-wrap md:flex-nowrap mt-5">
            <div className="grow mt-5 md:mr-5">
              <h3 className="text-xl font-bold mb-5">{featureData.features.title}</h3>
              {featureData.features.features.map((feature) => (
                <Feature
                  key={feature.title}
                  icon={feature.icon}
                  title={feature.title}
                  description={feature.description}
                />
              ))}
            </div>
            <div className="grow mt-5 md:ml-5">
              <h3 className="text-xl font-bold mb-5">{featureData.proFeatures.title}</h3>
              {featureData.proFeatures.features.map((feature) => (
                <Feature
                  key={feature.title}
                  icon={feature.icon}
                  title={feature.title}
                  description={feature.description}
                />
              ))}
            </div>
          </div>
          <div className="grid place-items-center mt-5">
            <a href="http://itunes.apple.com/app/letter/id498506154">
              <Image src="/app_store_en.svg" alt="App Store" width={150} height={50} className="mt-5" />
            </a>
          </div>
        </Container>
        { articles && (
        <Container>
          <Heading id="blog">{t('blog.title')}</Heading>
          <Blog articles={articles} />
        </Container>
        )}
        <Container>
          <div className="flex flex-col items-center">
            <div className="w-full md:w-9/12">
              <Heading id="contact">{t('contact.title')}</Heading>
              <ContactForm />
            </div>
          </div>
        </Container>
      </Section>
    </Page>
  );
}
