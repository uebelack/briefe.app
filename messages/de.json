{
  "common": {
    "title": "Briefe App",
    "title_long": "Briefe App - Mühelos professionelle Briefe auf iPhone, iPad, Mac und Android-Geräten erstellen",
    "scroll_down": "Nach unten scrollen",
    "language": "Deutsch",
    "download": "Herunterladen",
    "android_only": "* Aktuell nicht verfügbar auf Android-Geräten",
    "ios_only": "** Aktuell nur auf iPhone und iPad verfügbar"
  },
  "home": {
    "teaser": "Im Handumdrehen professionell formatierte Briefe erstellen und drucken."
  },
  "details": {
    "title": "Details",
    "p1": "Selbst in einer Zeit digitaler Kommunikation und sofortiger Vernetzung bleiben traditionelle Briefe eine unverzichtbare Form des Austauschs. Warum? Weil sie eine zeitlose und authentische Art der Korrespondenz darstellen. Ob für persönliche oder geschäftliche Nachrichten – Briefe vermitteln eine Tiefe und Bedeutung, die in E-Mails oder Textnachrichten oft verloren geht. Sie sind ein Ausdruck von Sorgfalt und Aufmerksamkeit und hinterlassen einen bleibenden Eindruck. Unsere Brief-App kombiniert diese zeitlose Tradition mit modernem Komfort und ermöglicht es Ihnen, professionelle Briefe einfach zu erstellen, zu teilen und zu verwalten.",
    "p2": "Die Briefe App ermöglicht es Ihnen, professionell formatierte Briefe schnell und mühelos auf iPhone, iPad, Mac und Android-Geräten zu erstellen und zu drucken. Ihre Briefe erhalten ein ansprechendes Layout, das einen professionellen Eindruck hinterlässt. Mit nur einem Klick können Sie Ihre Briefe direkt aus der App drucken. Sie haben auch die Möglichkeit, Ihre Briefe digital zu teilen oder als PDF zu exportieren, um sie auf verschiedenen Plattformen zu verteilen. Die App synchronisiert sich nahtlos über iCloud auf all Ihren Geräten, sodass Sie jederzeit und überall auf Ihre Briefe zugreifen können. Dank der einfachen Suchfunktion lassen sich bereits verfasste Briefe schnell wiederfinden.",
    "pro_title": "Mit Pro die KI-gestützte Briefassistenz freischalten",
    "p3": "Steigen Sie auf die Pro-Version um und heben Sie Ihr Briefschreiben auf ein neues Level mit unserem KI-Assistenten! Optimieren Sie mühelos Struktur, Klarheit und Ton Ihrer Briefe, während die ursprüngliche Intention erhalten bleibt. Egal, ob Sie einen Entwurf verbessern, einen vollständigen Brief nur anhand eines Betreffs generieren oder einfach Inspiration suchen – die KI sorgt für professionelle und perfekt formulierte Ergebnisse. Sie unterstützt mehrere Sprachen und passt sich Ihrem Schreibstil an, um Ihre Korrespondenz noch einfacher zu gestalten.",
    "p4": "Die Pro-Version bietet zudem weitere Premium-Funktionen wie das Hinzufügen persönlicher Unterschriften direkt in der App, das mühelose Importieren von Kontaktadressen aus Ihrem Adressbuch und eine große Auswahl an Vorlagen für verschiedene Anlässe. Darüber hinaus können Sie das Format Ihrer Briefe individuell anpassen.",
    "p5": "Die Briefe App vereinfacht Ihre Korrespondenz und ermöglicht es Ihnen, Briefe mühelos zu erstellen, zu teilen und zu drucken – und das mit Stil, Klarheit und Benutzerfreundlichkeit."
  },
  "features": {
    "title": "Funktionen",
    "description": "Um Ihnen einen umfassenden Überblick zu verschaffen, haben wir alle Funktionen im Detail aufgelistet, und zwar entsprechend ihrer Verfügbarkeit in den verschiedenen App-Versionen. So können Sie leicht erkennen, welche Features jede Version der App bietet.",
    "format": {
      "title": "Automatische Formatierung",
      "description": "Briefe werden automatisch professionell formatiert"
    },
    "print": {
      "title": "Drucken",
      "description": "Drucken Sie Ihre Briefe bequem direkt aus der App"
    },
    "export": {
      "title": "Teilen & Exportieren",
      "description": "Teilen Sie Ihre Briefe digital oder exportieren Sie sie als PDF-Dateien"
    },
    "sync": {
      "title": "Speicherung",
      "description": "Ihre Briefe werden über die iCloud geräteübergreifend synchronisiert"
    },
    "signatures": {
      "title": "Unterschriften",
      "description": "Fügen Sie persönliche Unterschriften direkt in der App hinzu"
    },
    "contacts": {
      "title": "Kontakte",
      "description": "Verwenden Sie Ihre Kontaktadressen direkt aus Ihrem Adressbuch"
    },
    "templates": {
      "title": "Vorlagen",
      "description": "Nutzen Sie eine breite Palette von Vorlagen für verschiedene Gelegenheiten"
    },
    "layout": {
      "title": "Layout",
      "description": "Passen Sie die Formatierung Ihrer Briefe individuell an Ihre Bedürfnisse an"
    }
  },
  "faq": {
    "menu": "FAQ",
    "title": "Häufig gestellte Fragen"
  },
  "contact": {
    "title": "Kontakt",
    "subtitle": "Bitte kontaktieren Sie mich, wenn Sie Fragen haben",
    "email_placeholder": "Ihre E-Mail-Adresse",
    "subject_placeholder": "Ihr Anliegen",
    "message_placeholder": "Ihre Nachricht",
    "privacy_1": "Ich akzeptiere die",
    "privacy_2": "Datenschutzerklärung",
    "privacy_link": "/de/datenschutz",
    "privacy_info": "Ihre E-Mail-Adresse und Ihre Nachricht werden gespeichert, damit ich darauf antworten kann. Ihre Daten werden nicht für andere Zwecke verwendet.",
    "submit": "Senden",
    "email_error": "Bitte geben Sie eine gültige E-Mail-Adresse ein",
    "subject_error": "Bitte geben Sie Ihr Anliegen ein",
    "message_error": "Bitte geben Sie eine Nachricht ein",
    "captcha_error": "Bitte lösen Sie das Captcha",
    "privacy_error": "Bitte akzeptieren Sie die Datenschutzerklärung",
    "success": "Vielen Dank für Ihre Nachricht! Wir werden uns so schnell wie möglich bei Ihnen melden!",
    "error": "Leider ist ein unerwarteter Fehler aufgetreten. Bitte versuchen Sie es später erneut oder senden Sie uns eine E-Mail an support@briefe.app.",
    "cookie": "Bitte akzeptieren Sie die Cookies am unteren Rand des Bildschirms, um eine Nachricht zu übermitteln oder senden Sie eine E-Mail an support@briefe.app."
  },
  "footer": {
    "rights": "Alle Rechte vorbehalten.",
    "imprint": "Impressum",
    "imprint_link": "/de/impressum",
    "privacy": "Datenschutzerklärung",
    "privacy_link": "/de/datenschutz"
  },
  "cookie_consent": {
    "message": "Diese Webseite verwendet Cookies, um Ihnen einen besseren Service bieten zu können. Um mehr zu erfahren, lesen Sie bitte unsere",
    "accept": "Akzeptieren"
  },
  "lost": {
    "title": "Verloren!? 404 - Nicht gefunden",
    "description": "Die Seite, die Sie suchen, existiert nicht.",
    "button": "Startseite"
  },
  "help": {
    "menu": "Hilfe",
    "link": "hilfe",
    "title": "Hilfe",
    "how_to_title": "So schreiben Sie einen Brief mit der Briefe-App",
    "how_to_001": "Um mit dem Verfassen Ihres Briefes zu beginnen, tippen Sie einfach auf die Plus-Schaltfläche in der oberen rechten Ecke des Hauptbildschirms. Sobald Sie auf die Schaltfläche getippt haben, öffnet sich ein neuer Bildschirm, in dem Sie Ihren Brief ganz einfach verfassen können.",
    "how_to_002": "Geben Sie Absender- und Empfängeradresse ein, fügen Sie einen Betreff hinzu und verfassen Sie Ihre Nachricht im Feld 'Nachricht'. Sobald Sie Ihren Brief fertiggestellt haben, tippen Sie auf 'Speichern' in der oberen rechten Ecke.",
    "how_to_003": "Ihr Brief wird automatisch formatiert und angezeigt. Nun können Sie ihn entweder ausdrucken, indem Sie auf das Druckersymbol tippen, oder ihn über die Schaltfläche 'Teilen' (das quadratische Symbol mit einem Pfeil) per E-Mail versenden."
  },
  "blog": {
    "menu": "Blog",
    "title": "Blog - Gedanken, Geschichten und Ideen"
  },
  "call_to_action": {
    "text": "Laden Sie jetzt die kostenlose Briefe App herunter und schreiben Sie Ihren ersten Brief!"
  }
}