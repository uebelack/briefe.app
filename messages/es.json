{
  "common": {
    "title": "Aplicación de Cartas",
    "title_long": "Aplicación de Cartas - Crea cartas profesionales sin esfuerzo en iPhone, iPad, Mac y dispositivos Android",
    "scroll_down": "Desplázate hacia abajo",
    "language": "Español",
    "download": "Descargar",
    "android_only": "* actualmente no disponible en dispositivos Android",
    "ios_only": "** actualmente sólo en iPhone y iPad"
  },
  "home": {
    "teaser": "Crea e imprime cartas con formato perfecto en poco tiempo."
  },
  "details": {
    "title": "Detalles",
    "p1": "Incluso en una era dominada por la comunicación digital y la conectividad instantánea, las cartas tradicionales siguen siendo una forma esencial de correspondencia. ¿Por qué? Porque representan un modo de comunicación atemporal y auténtico. Ya sea para mensajes personales o profesionales, las cartas transmiten una profundidad y significado que a menudo se pierde en los correos electrónicos o mensajes de texto. Son una expresión de cuidado y atención, dejando una impresión duradera. La Aplicación de Cartas combina esta tradición con la comodidad moderna, permitiéndote crear, compartir y gestionar cartas profesionales de manera sencilla.",
    "p2": "La Aplicación de Cartas te permite redactar y imprimir cartas con un formato profesional de manera rápida y sencilla en iPhone, iPad, Mac y dispositivos Android. Tus cartas tendrán un diseño atractivo que dejará una impresión profesional. Con un solo clic, puedes imprimir directamente desde la app. También tienes la opción de compartir tus cartas digitalmente o exportarlas como archivos PDF para distribuirlas en diversas plataformas. La aplicación se sincroniza perfectamente a través de iCloud en todos tus dispositivos, permitiéndote acceder a tus cartas en cualquier momento y lugar. Gracias a su función de búsqueda intuitiva, puedes encontrar fácilmente cartas previamente redactadas.",
    "pro_title": "Desbloquea la Asistencia de Cartas con IA en la versión Pro",
    "p3": "¡Lleva tu redacción al siguiente nivel con nuestro Asistente de IA en la versión Pro! Mejora la estructura, claridad y tono de tus cartas de manera sencilla, manteniendo siempre la intención original. Ya sea que necesites perfeccionar un borrador, generar una carta completa a partir de un asunto o simplemente encontrar inspiración, la inteligencia artificial garantiza resultados profesionales y bien elaborados. Además, admite varios idiomas y se adapta a tu estilo de escritura para que la correspondencia sea más fácil y efectiva.",
    "p4": "La versión Pro también desbloquea funciones premium adicionales, como la posibilidad de añadir firmas personales directamente en la aplicación, importar direcciones de contacto desde tu libreta de direcciones y utilizar una amplia variedad de plantillas para distintas ocasiones. Además, puedes personalizar el formato de tus cartas según tus necesidades específicas.",
    "p5": "La Aplicación de Cartas simplifica tu correspondencia, permitiéndote crear, compartir e imprimir cartas de manera fácil, manteniendo siempre un estilo claro, profesional y fácil de usar."
  },
  "features": {
    "title": "Características",
    "description": "Para brindarte una visión completa, hemos enumerado todas las características en detalle, según su disponibilidad en las diferentes versiones de la aplicación. De esta manera, puedes ver fácilmente qué características ofrece cada versión de la aplicación.",
    "format": {
      "title": "Auto Formato",
      "description": "Las cartas se formatean automáticamente de manera profesional"
    },
    "print": {
      "title": "Imprimir",
      "description": "Imprime fácilmente tus cartas directamente desde la aplicación"
    },
    "export": {
      "title": "Compartir y Exportar",
      "description": "Comparte tus cartas de forma digital o exporta archivos PDF"
    },
    "sync": {
      "title": "Almacenamiento",
      "description": "Tus cartas se sincronizan en tus dispositivos usando iCloud"
    },
    "signatures": {
      "title": "Firmas",
      "description": "Agrega firmas personales directamente en la aplicación"
    },
    "contacts": {
      "title": "Contactos",
      "description": "Incorpora fácilmente direcciones de contacto desde tu libreta de direcciones"
    },
    "templates": {
      "title": "Plantillas",
      "description": "Utiliza una amplia gama de plantillas para diversas ocasiones"
    },
    "layout": {
      "title": "Diseño",
      "description": "Personaliza el diseño de tus cartas según tus necesidades específicas"
    }
  },
  "faq": {
    "menu": "FAQ",
    "title": "Preguntas más frecuentes"
  },
  "contact": {
    "title": "Contacto",
    "subtitle": "Por favor, contáctame si tienes alguna pregunta",
    "email_placeholder": "Tu dirección de correo electrónico",
    "subject_placeholder": "Tu asunto",
    "message_placeholder": "Tu mensaje",
    "privacy_1": "Acepto la",
    "privacy_2": "política de privacidad",
    "privacy_link": "/es/privacy",
    "privacy_info": "Tu dirección de correo electrónico y mensaje se almacenarán para que pueda responder. Tus datos no se utilizarán para otros fines.",
    "submit": "Enviar",
    "email_error": "Por favor, ingresa una dirección de correo electrónico válida",
    "subject_error": "Por favor, ingresa un asunto",
    "message_error": "Por favor, ingresa un mensaje",
    "captcha_error": "Por favor, resuelve este captcha",
    "privacy_error": "Por favor, acepta la política de privacidad",
    "success": "¡Gracias por tu mensaje! Nos pondremos en contacto contigo lo antes posible.",
    "error": "Lamentablemente, ocurrió un error inesperado. Por favor, inténtalo de nuevo más tarde o envíanos un correo electrónico a support@letter-app.com.",
    "cookie": "Por favor, acepta las cookies en la parte inferior de la pantalla para enviar un mensaje o enviar un correo a support@letter-app.com."
  },
  "footer": {
    "rights": "todos los derechos reservados.",
    "imprint": "Aviso Legal",
    "imprint_link": "/es/imprint",
    "privacy": "Política de Privacidad",
    "privacy_link": "/es/privacy"
  },
  "cookie_consent": {
    "message": "Este sitio web utiliza cookies para garantizar que obtengas la mejor experiencia en nuestro sitio web. Para obtener más información, consulta nuestra",
    "accept": "Aceptar"
  },
  "lost": {
    "title": "¡Estás perdido! 404 - No encontrado",
    "description": "La página que buscas no existe",
    "button": "Ir a Inicio"
  },
  "help": {
    "menu": "Ayuda",
    "link": "ayuda",
    "title": "Ayuda",
    "how_to_title": "Cómo escribir una carta con la aplicación de cartas",
    "how_to_001": "Para empezar a redactar tu carta, simplemente pulsa el botón más en la esquina superior derecha de la pantalla principal. Una vez que hayas pulsado el botón, se abrirá una nueva pantalla en la que podrás redactar fácilmente tu carta",
    "how_to_002": "Introduce la dirección del remitente y del destinatario, añade un asunto y redacta tu mensaje en el campo \"Mensaje\". Una vez que haya terminado su carta, pulse \"Guardar\" en la esquina superior derecha",
    "how_to_003": "Su carta será formateada y mostrada automáticamente. Ahora puede imprimirla pulsando el icono de la impresora o enviarla por correo electrónico utilizando el botón \"Compartir\" (el icono cuadrado con una flecha)"
  },
  "templates": {
    "menu": "Plantillas",
    "title": "Plantillas"
  },
  "blog": {
    "menu": "Blog",
    "title": "Blog - Pensamientos, historias e ideas"
  },
  "call_to_action": {
    "text": "Descarga la Aplicación de Cartas ahora y empieza a escribir tu primera carta — ¡es gratis!"
  }
}