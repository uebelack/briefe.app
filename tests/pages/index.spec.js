import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import messages from '../../i18n';
import Home from '../../pages/index';

const useRouter = jest.spyOn(require('next/router'), 'useRouter');

useRouter.mockImplementation(() => ({
  pathname: '/',
}));

describe('<Home/>', () => {
  it('should render', async () => {
    const { container } = render(
      <Home />,
    );
    expect(container).toMatchSnapshot();
  });
});
