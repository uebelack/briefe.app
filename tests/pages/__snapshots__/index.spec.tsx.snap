// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> should render 1`] = `
<div>
  <div
    class="page"
  >
    <header
      class="fixed top-0 w-full bg-blue text-white"
    >
      <div
        class="md:container px-4 my-4 fade-in flex flex-col md:flex-row"
      >
        <a
          class="flex items-center transition ease-in-out delay-150 flex-grow"
          href="/"
        >
          <img
            alt="Letter logo"
            data-nimg="1"
            decoding="async"
            height="35"
            loading="lazy"
            src="/logo.svg"
            style="color: transparent;"
            width="50"
          />
          <div
            class="text-2xl ml-2 text-white"
          >
            Letter App
          </div>
        </a>
        <ul
          class="flex md:pl-1 pt-1 md:pt-1.5 "
        >
          <li
            class="mr-2"
          >
            <a
              href="/#details"
            >
              Details
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/#features"
            >
              Features
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/#contact"
            >
              Contact
            </a>
          </li>
          <li>
            <a
              class="pr-1"
              href="/en"
            >
              en
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <a
              class="pl-1"
              href="/de"
            >
              de
            </a>
          </li>
        </ul>
      </div>
    </header>
    <div
      class="grid place-items-center first bg-blue text-white section"
    >
      <div
        class="grid place-items-center md:container px-4"
      >
        <img
          alt="Letter app"
          data-nimg="1"
          decoding="async"
          height="605"
          loading="lazy"
          src="/devices_en.png"
          style="color: transparent;"
          width="992"
        />
        <a
          href="http://itunes.apple.com/app/letter/id498506154"
        >
          <img
            alt="App Store"
            class="mt-5"
            data-nimg="1"
            decoding="async"
            height="50"
            loading="lazy"
            src="/app_store_en.svg"
            style="color: transparent;"
            width="150"
          />
        </a>
        <div
          class="p-3 md:w-4/5 pt-10"
        >
          <div
            class="text-2xl md:text-3xl mb-4"
          >
            C
          </div>
        </div>
      </div>
    </div>
    <button
      aria-label="Scroll down"
      class="scroll-button"
      type="button"
    />
    <div
      class="section"
    >
      <div
        class="md:container px-4"
      >
        <div
          class="flex"
        >
          <div
            class="prose proese-briefe"
          >
            <h1
              class="text-2xl pt-5 mb-6"
              id="details"
            >
              Details
            </h1>
            <p>
              The Letter App allows you to quickly and effortlessly create and print professionally formatted letters. Your letters will have an appealing layout that leaves a professional impression. With a simple click, you can print your letters directly from the app. You also have the option to share your letters digitally or export them as PDF files to distribute them on various platforms. The app syncs seamlessly across your devices through iCloud, allowing you to access your letters anytime and anywhere. Thanks to the straightforward search function, you can easily retrieve previously composed letters.
            </p>
            <p>
              In the Pro version, you'll have access to additional features, such as adding personal signatures directly within the app, effortlessly incorporating contact addresses from your address book, and utilizing a wide range of templates for various occasions. You can customize settings to adapt the format of your letters to your specific needs. The Letter App simplifies your correspondence, enabling you to create, share, and print letters with ease, all while maintaining style and user-friendliness.
            </p>
          </div>
          <div
            class="pt-32 ml-20 hidden lg:inline"
          >
            <img
              alt="Letter app"
              data-nimg="1"
              decoding="async"
              height="450"
              loading="lazy"
              src="/details_en.png"
              style="color: transparent;"
              width="225.5"
            />
          </div>
        </div>
      </div>
      <div
        class="md:container px-4"
      >
        <h1
          class="text-2xl pt-5 mb-6"
          id="features"
        >
          Features
        </h1>
        <p>
          To give you a comprehensive overview, we have listed all the features in detail, according to their availability in the different app versions. This way, you can easily see which features each version of the app offers.
        </p>
        <div
          class="flex flex-wrap md:flex-nowrap mt-5"
        >
          <div
            class="grow mt-5 md:mr-5"
          >
            <h3
              class="text-xl font-bold mb-5"
            >
              Free Version
            </h3>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üìÑ
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Auto Format
                </div>
                <div
                  class="text-lg"
                >
                  Letters are automatically formatted in a professional manner
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üñ®Ô∏è
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Print
                </div>
                <div
                  class="text-lg"
                >
                  Easily print your letters directly from the app
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üì§
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Share & Export
                </div>
                <div
                  class="text-lg"
                >
                  Share your letters digitally or export them as PDF files
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                ‚òÅÔ∏è
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Storage
                </div>
                <div
                  class="text-lg"
                >
                  Your letters are synchronized across devices using iCloud
                </div>
              </div>
            </div>
          </div>
          <div
            class="grow mt-5 md:ml-5"
          >
            <h3
              class="text-xl font-bold mb-5"
            >
              PRO Version
            </h3>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                ‚úçÔ∏è
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Signatures
                </div>
                <div
                  class="text-lg"
                >
                  Add personal signatures directly within the app
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üìá
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Contacts
                </div>
                <div
                  class="text-lg"
                >
                  Easily incorporate contact addresses from your address book
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üìã
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Templates
                </div>
                <div
                  class="text-lg"
                >
                  Utilize a wide range of templates for various occasions
                </div>
              </div>
            </div>
            <div
              class="flex mb-5"
            >
              <div
                class="text-4xl mr-5"
              >
                üìê
              </div>
              <div>
                <div
                  class="text-lg font-bold"
                >
                  Layout
                </div>
                <div
                  class="text-lg"
                >
                  Customize the layout of your letters to your specific needs
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="grid place-items-center mt-5"
        >
          <a
            href="http://itunes.apple.com/app/letter/id498506154"
          >
            <img
              alt="App Store"
              class="mt-5"
              data-nimg="1"
              decoding="async"
              height="50"
              loading="lazy"
              src="/app_store_en.svg"
              style="color: transparent;"
              width="150"
            />
          </a>
        </div>
      </div>
      <div
        class="md:container px-4"
      >
        <div
          class="flex flex-col items-center"
        >
          <div
            class="w-full md:w-9/12"
          >
            <h1
              class="text-2xl pt-5 mb-6"
              id="contact"
            >
              Contact
            </h1>
            <div
              class="lazyload-wrapper "
            >
              <div
                class="lazyload-placeholder"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="md:container mt-20 mb-20 text-center"
    >
      <div
        class="mb-2 px-4 my-4"
      >
        ¬©
         
        2023
         
        √úbelacker Solutions GmbH
        <br />
        <a
          href="/en/imprint"
        >
          Imprint
        </a>
         
        |
         
        <a
          href="/en/privacy"
        >
          Privacy Policy
        </a>
      </div>
    </footer>
  </div>
</div>
`;
